### ISSUE TITLE
Op request: layer normalization

### ISSUE THREAD

**Issue by gaikwadrahul8** [google-ai-edge/LiteRT#173]
Layer normalization is available in tensorflow https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization
It is not part of the tflite supported ops
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/builtin_ops.h

I would like to request support for this op. Please let me know if it is in development or scoped for future release.

**Comment by gaikwadrahul8** [comment]
This issue originally reported by @vk1214  has been moved to this dedicated repository for [LiteRT](https://github.com/google-ai-edge/LiteRT/) to enhance issue tracking and prioritization. To ensure continuity, we have created this new issue on your behalf.

We appreciate your understanding and look forward to your continued involvement.

**Comment by pkgoogle** [comment]
Original Issue: https://github.com/tensorflow/tensorflow/issues/60532

**Comment by aravindmurali22** [comment]
Hi @vk1214 , thank you for this feature suggestion.

As the project has evolved, our development priorities have shifted to focus on the current roadmap. While this is a valuable suggestion, it does not align with our immediate plans.

We are closing this issue to ensure our tracker accurately reflects our active development queue. We appreciate your input and hope you'll continue to share your ideas with us.
